priv/repo/migrations/20240625211759_remove_enable_authorization_flag.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Realtime.Repo.Migrations.RemoveEnableAuthorizationFlag do
+  use Ecto.Migration
+
+  def change do
+    alter table(:tenants) do
+      remove :enable_authorization
+    end
+  end
+end
```

Comment:
Need to call it in Realtime.Tenants.Migrations, no?

Reply:
no, this are on realtime itself